Makefile

```diff
@@ -197,6 +197,9 @@ ifeq (, $(shell which ginkgo))
 	$(call go-get-tool,$(GINKGO),github.com/onsi/ginkgo/ginkgo@v1.16.5)
 endif
 
+validate-swagger: ## Validate swagger
+	$(DOCKER) run -it --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml -v $(PWD)/swagger.yaml:/tmp/swagger.yaml stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml 
```

Comment:
pls remove trailing whitespace

Reply:
don't think it is needed here. the file should exist only inside a container that will be removed after the execution ends.